jupyterhub/templates/proxy/deployment.yaml

```diff
@@ -44,6 +45,12 @@ spec:
                     - key: release
                       operator: In
                       values: [{{ .Release.Name | quote }}]
+      {{- if $manualHTTPSwithsecret }}
+      volumes:
+        - name: tls-secret
+          secret:
+            secretName: {{ .Values.proxy.https.secret }}
```

Comment:
Can you fill out values.yaml with null values for these?

I'm guessing it should look like:

```yaml
proxy:
  https:
    type: secret
    secret:
      name: 'my-tls-secret'
      key: 'mydomain.key'
      cert: 'mydomain.crt'
```

Does that look right?

Reply:
Yes, the only thing would be that type: manualsecret, but that's easily transformable to secret, I can do that if you want !